docs_theme/css/bootstrap.css

```diff
@@ -850,7 +850,7 @@ address {
 code,
 pre {
   padding: 0 3px 2px;
-  font-family: Monaco, Menlo, Consolas, "Courier New", monospace;
+  font-family: monospace, "Courier New", Monaco, Menlo, Consolas;
```

Comment:
This makes the fallback monospace font the first to be selected right? Surely the idea is the fallback goes last?